modules/oxide/src/less/theme/components/form/textfield.less

```diff
@@ -21,7 +21,7 @@
 @textfield-disabled-text-color: fade(@textfield-text-color, 85%);
 
 @textfield-focus-background-color: @textfield-background-color;
-@textfield-focus-border-color: @color-tint;
+@textfield-focus-border-color: #2d6adf;
```

Comment:
This should likely gets it's value from somewhere else not be specific to the textfield.

Reply:
I noticed this after making the PR, since I inherited the branch. I made a commit to revert some of these unnecessary changes but was trying to fix a test failure before pushing. I've pushed it now but I'll re-request your review once the tests are passing.